yoga/Yoga.h

```diff
@@ -219,7 +219,8 @@ YG_NODE_LAYOUT_EDGE_PROPERTY(float, Border);
 YG_NODE_LAYOUT_EDGE_PROPERTY(float, Padding);
 
 WIN_EXPORT void YGSetLogger(YGLogger logger);
-WIN_EXPORT void YGLog(YGLogLevel level, const char *message, ...);
+WIN_EXPORT void YGLog(const YGNodeRef node, YGLogLevel level, const char *message, ...);
+WIN_EXPORT void YGLogWithConfig(const YGConfigRef config, YGLogLevel level, const char *format, ...);
```

Comment:
`YGLogWithConfig` is required for the places were we don't have node, just a config, e.g. `YGNodeNewWithConfig` and `YGConfigSetPointScaleFactor``